core/cli/src/params.rs

```diff
@@ -227,63 +225,81 @@ pub struct TransactionPoolParams {
 pub struct ExecutionStrategies {
 	/// The means of execution used when calling into the runtime while syncing blocks.
 	#[structopt(
-		long = "syncing-execution",
+		long = "execution-syncing",
 		value_name = "STRATEGY",
 		raw(
 			possible_values = "&ExecutionStrategy::variants()",
 			case_insensitive = "true",
 			default_value = r#""NativeElseWasm""#
 		)
 	)]
-	pub syncing_execution: ExecutionStrategy,
+	pub execution_syncing: ExecutionStrategy,
 
 	/// The means of execution used when calling into the runtime while importing blocks.
 	#[structopt(
-		long = "importing-execution",
+		long = "execution-import-block",
```

Comment:
imo it's a bit weird to have the order like this in the flag `--execution-block-import`, can we keep the originals?

Reply:
First I thought that these `long` names would determine the order in the cli, after that I realized that actually the field name determines the order. So, I could revert this. However, I think with this order they look more uniform and the `execution` is more like a namespace.